### ISSUE TITLE
Drag and drop to meet accessibility requirements

### ISSUE THREAD

**Issue by discodavey** [libero/editor#498]
## Description
Draggable interface elements are not accessible using keyboard controls or screen readers. JAWS screen reader works fine. 

## Tasks
- [ ] Draggable list items are to work in a similar way to the code sandbox example below
- [ ] NVDA can read content but cannot move list items. Research how lists can be moved using NVDA reader
- [ ] Carry out necessary development work based on point 2 if needed
- [ ] Screen reader instructional text to be provided by designer

## DoD
- [ ] Drag and drop feature work using keyboard controls
- [ ] Both screen readers to read out all possible action on draggable lists
- [ ] Keyboard drag and drop to enabled for sortable lists using NVDA screen reader

## Reference
- [Code sandbox keyboard drag/drop example](https://codesandbox.io/s/o104x95y86?file=/src/index.js)

**Comment by NickDuf** [comment]
@discodavey would you mind taking a look at the following articles and proposing some tasks? I'm happy to describe the dod.
